infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/BridgeErrorInstance.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.service.smartevents.infra.exceptions;
+
+import java.util.UUID;
+
+public class BridgeErrorInstance extends BridgeError {
+
+    private final String uuid;
```

Comment:
@manstis This doesn't feel specific to errors from the Shard IMHO. If we think there is value in something like then I think we should apply it equally to all Errors, perhaps as something like a `request_id` or `operation_id` field on the Error.

I like the idea, but I wonder if we should consider it within the scope of tracing and perhaps outside this PR so that we can apply consistently all scenarios.

Your thoughts?

cheers,
Rob

Reply:
It's not used specifically for errors from the Shard; it's also used for errors from our [`Worker`'s](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/968/files#diff-1319ec61fecd04bc61a3e44b2bffba77efe39a0885cf8d585aa9b01b97736859R218-R225) in the Manager. It is used to help identify a specific occurrence of an error in the server logs. User A may perform an operation that causes Error X; so may User B or C. The _instance_ class contains a UUID to identify a specific occurrence of Error X in the logs. 

Other meta-data _could_ be useful (for other scenarios.. what IDK). 

Yes, it does overlap with Tracing but Tracing is a long way away (and we'd need to include the Trace context in the message to Users too; to be sure SysOps/DevOps/whoever looks at the right bit occurrence in the logs.. would we not?)